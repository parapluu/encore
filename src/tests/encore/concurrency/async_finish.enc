class Main
  def main(): void {
    finish {
      print 23;
      async(print "Tasks are awesome")
    };
    print "Leave finish";
  }
