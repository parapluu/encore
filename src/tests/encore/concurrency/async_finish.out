23
Tasks are awesome
Leave finish
